MDANSE/Src/MDANSE/MolecularDynamics/Trajectory.py

```diff
@@ -418,11 +427,13 @@ def create_average_atom(
                 [database.get_atom_property(element_name, property), element_count]
             )
         if property in additive_atom_properties:
-            total = np.sum([float(x[0]) * int(x[1]) for x in temp])
+            total = np.sum([complex(x[0]) * int(x[1]) for x in temp])
+            total = str(total)
         elif property in averaged_atom_properties:
-            total = np.sum([float(x[0]) * int(x[1]) for x in temp]) / np.sum(
+            total = np.sum([complex(x[0]) * int(x[1]) for x in temp]) / np.sum(
                 [int(x[1]) for x in temp]
             )
```

Comment:
I have a feeling instantiating the list will be worse than iterating. 
```suggestion
            total = sum(complex(x[0]) * int(x[1]) for x in temp) / sum(
                int(x[1]) for x in temp
            )
```